components/atoms/ToggleSwitch/toggle-switch.tsx

```diff
@@ -1,27 +1,33 @@
 import * as Switch from "@radix-ui/react-switch";
 import clsx from "clsx";
 
-interface ToggleSwitchProps {
+type ToggleSwitchProps = {
```

Comment:
The props look a little more complex, but basically it's so that you can either have `ariaLabel` or `ariaLabelledBy`, not both.